CHANGES_NEXT_RELEASE

```diff
@@ -4,3 +4,25 @@
 - Fix: fix issue with unrecognized NGSILD-Tenant and NGSILD-Path headers
 - Fix: mongodb dependency moved to devDependencies (as it is used only in test code)
 - Upgrade mongodb dep from 3.6.12 to 4.7.0
+- Upgrade body-parser dependency from 1.19.0 to 1.19.1
+- Upgrade express dependency from 4.16.4 to 4.17.2
+- Upgrade got dependency from 11.8.2 to 11.8.3
+- Upgrade moment-timezone dependency from 0.5.25 to 0.5.34
+- Upgrade mongoose dependency from 5.7.7 to 5.13.14
+- Upgrade query-string dependency from 6.5.0 to 7.1.1
+- Upgrade underscore dependency from 1.12.1 to 1.13.2
+- Upgrade uuid dependency from 3.3.2 to 8.3.2
+- Upgrade coveralls dev dependency from 3.1.0 to 3.1.1
+- Upgrade eslint dev dependency from 7.5.0 to 8.8.0
+- Upgrade eslint-config-tamia dev dependency from 7.2.5 to 7.2.7
+- Upgrade eslint-plugin-prettier dev dependency from 3.1.4 to 4.0.0
+- Upgrade lint-staged dev dependency from 10.2.11 to 12.3.3
+- Upgrade prettier dev dependency from 2.0.5 to 2.5.1
+- Upgrade mocha dev dependency from 8.0.1 to 9.2.0
+- Upgrade nock dev dependency from 13.0.3 to 13.2.4
```

Comment:
```suggestion
- Upgrade nock dev dependency from 13.0.3 to 13.2.5
```

Reply:
I've just gone through the deps again e870675 and removed two more high vulnerabilities.  Nock should be at 13.2.7